package.json

```diff
@@ -27,7 +27,7 @@
     "concourse-build-electron": "make webpack",
     "concourse-test": "npm test",
     "concourse-test-electron": "npm test",
-    "concourse-test-node-cli": "make lint test-cli sanity-checks"
+    "concourse-test-node-cli": "make lint-js lint-cpp lint-spell test-cli"
```

Comment:
Might make sense to add a phoney target (e.g. `lint-cli`) to the Makefile which depends on `lint-js lint-cpp lint-spell` ?
Why was `sanity-checks` removed?